content/docs/authorization/release/_index.md

```diff
@@ -23,4 +22,6 @@ Description: >
 - Update Authorization documentation with supported versions of k3s-selinux and container-selinux packages. ([#393](https://github.com/dell/csm/issues/393))
 - Using Authorization without dependency on jq. ([#390](https://github.com/dell/csm/issues/390))
 - Authorization Documentation Improvement. ([#384](https://github.com/dell/csm/issues/384))
-- Unit test failing for csm-authorization. ([#382](https://github.com/dell/csm/issues/382))
\ No newline at end of file
+- Unit test failing for csm-authorization. ([#382](https://github.com/dell/csm/issues/382))
+- Karavictl has incorrect permissions after download. ([#360](https://github.com/dell/csm/issues/360 [github.com])) 
+- Helm deployment of Authorization denies a valid request path from csi-powerflex. ([#353](https://github.com/dell/csm/issues/353 [github.com]))
```

Comment:
Ensure these links still work. I see some have [github.com] which may make the link invalid